### DISCUSSION TITLE
Path does not have our version

### DISCUSSION THREAD

**Discussion by PenguinMoon** [community/community#24069] [Apps, API and Webhooks]
<p>Im running into an error with GitHub desktop that I need help with.</p>
<p>Im getting the error:</p>
<p>“error: path ‘Assets/Scripts/FrameRateLimiter.cs’ does not have our version”</p>
<p>I know this is due to this file having version problems, but I don’t know how to fix this neither the local copy or copy on git work.</p>
<p>Any help would be much appreciated.</p>

**Comment by Wabri** [comment]
<p>Hi,</p>
<p>Could you please give more infos? Or a screenshot is even better!</p>

**Comment by PenguinMoon** [comment, marked as answer]
<p>Oh sorry, I should have posted earlier. I found a fix. I had to discard the files that were causing issues and then it was able to successfully merge.</p>
